great_expectations/init_notebooks/pandas/validation_playground.ipynb

```diff
@@ -64,9 +68,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 3. Pick a csv and set the expectation suite name\n",
+    "## 3. Pick a csv and the expectation suite\n",
     "\n",
-    "Internally, Great Expectations represents csvs and dataframes as `DataAsset`s and uses this notion to link them to `Expectation Suites`. To learn more about `DataAssets` and how their names are built, see [the reference](https://docs.great_expectations.io/en/latest/reference/data_context_reference.html#data-asset-names). "
+    "Internally, Great Expectations represents csvs and dataframes as `Data Asset`s and uses this notion to link them to `Expectation Suites`. \n",
+    "\n",
+    "[Read more in the tutorial](https://docs.greatexpectations.io/en/latest/tutorials/validate_data.html?utm_source=notebook&utm_medium=validate_data#pick-a-data-asset-and-and-expectation-suite)\n"
```

Comment:
I intentionally compacted these newlines to make the notebook less overwhelming. Shorter is better.

Reply:
Also remove the space in `Data Asset`

Reply:
+1